relayer-cli/src/commands/update.rs

```diff
@@ -1,8 +1,94 @@
 //! `update` subcommand
 
 use abscissa_core::{Command, Help, Options, Runnable};
+use ibc::events::IbcEvent;
+use ibc::ics02_client::client_state::ClientState;
+use ibc::ics24_host::identifier::{ChainId, ClientId};
+use ibc::NonZeroHeight;
+use ibc_relayer::foreign_client::ForeignClient;
 
-use crate::commands::tx::client::TxUpdateClientCmd;
+use crate::application::app_config;
+use crate::cli_utils::spawn_chain_runtime;
+use crate::conclude::Output;
+
+#[derive(Clone, Command, Debug, Options)]
+pub struct UpdateClientCmd {
+    #[options(free, required, help = "identifier of the destination chain")]
+    dst_chain_id: ChainId,
+
+    #[options(
+        free,
+        required,
+        help = "identifier of the client to be updated on destination chain"
+    )]
+    dst_client_id: ClientId,
+
+    #[options(help = "the target height of the client update", short = "h")]
+    target_height: Option<u64>,
```

Comment:
Just FYI: we may need to update the guide ([this section](https://hermes.informal.systems/commands/path-setup/clients.html#update-client)) with this change.

Reply:
definitely needs guide update. Also, if we want to change this as compared to `tx raw` we should discuss to understand why and how. At first look I would propose `hermes client update <chain-id> <cliend-id>` (i.e. drop the target_height)